osquery/tables/system/windows/userassist.cpp

```diff
@@ -0,0 +1,143 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/tables/system/windows/registry.h>
+#include <osquery/utils/conversions/split.h>
+#include <osquery/utils/system/time.h>
+#include <string>
+
+namespace osquery {
+namespace tables {
+
+// Decode ROT13 sub key value
+std::string rot_decode(std::string value_key_reg) {
```

Comment:
It may have been mentioned before but we should include a unit test specifically for this function.

Reply:
i added an additional test for this specific function. Let me know if its ok or if ur expecting some else.
Just to add isnt this function already being tested when executing this query in the test file?
If the function fails the ROT13 data wont decode properly to "Microsoft.Windows.Explorer"
`"select * from userassist where path is 'Microsoft.Windows.Explorer'"`